docs/core-concepts/Cli.md

```diff
@@ -0,0 +1,142 @@
+# CrewAI CLI Documentation
+
+The CrewAI CLI provides a set of commands to interact with CrewAI, allowing you to create, train, run, and manage crews and pipelines.
+
+## Installation
```

Comment:
_Suggestion_: Great work, would be nice to add "pip install poetry " as requirement since our CLI is so heavily dependent